extensions/skyportal/skyportal/handlers/api/kowalski_filter.py

```diff
@@ -352,11 +352,30 @@ def patch(self, filter_id):
                 "radius",
                 "implements_update",
                 "ignore_allocation_ids",
+                "not_if_tns_reported",
             }
             if not set(auto_followup.keys()).issubset(valid_keys):
                 return self.error(
                     f"auto_followup dict keys must be a subset of {valid_keys}"
                 )
+            if "not_if_tns_reported" in auto_followup:
+                # if it's not empty or None, verify it's a a float or an int
+                if (
+                    auto_followup["not_if_tns_reported"] is not None
+                    and str(auto_followup["not_if_tns_reported"]).strip() != ""
+                ):
+                    try:
```

Comment:
Maybe also worth making note of the expected units here.

Reply:
Good point, I'll add that in before merging, thanks!